.github/workflows/tests_linux.yml

```diff
@@ -368,184 +396,62 @@ jobs:
-        run: |
-            cmake . -BBuildCov -DCMAKE_BUILD_TYPE=Debug -DENABLE_BLAS=ON -DENABLE_PYTHON=OFF -DENABLE_KOKKOS=ON -DCMAKE_PREFIX_PATH=${{ github.workspace }}/Kokkos -DBUILD_TESTS=ON -DENABLE_COVERAGE=ON -DCMAKE_CXX_COMPILER="$(which g++-$GCC_VERSION)" -G Ninja
-            cmake --build ./BuildCov
-            cd ./BuildCov
-            ./pennylane_lightning_test_runner
-            lcov --directory . -b ../pennylane_lightning/src --capture --output-file coverage.info
-            lcov --remove coverage.info '/usr/*' --output-file coverage.info
-            mv coverage.info coverage-${{ github.job }}.info
-
-      - name: Upload code coverage results
-        uses: actions/upload-artifact@v3
-        with:
-          name: ubuntu-codecov-results-cpp
-          path: ./BuildCov/coverage-${{ github.job }}.info
-
-  pythontestswithKokkosAndOpenBLAS:
-    needs: [build_and_cache_Kokkos]
-    strategy:
-      matrix:
-        os: [ubuntu-22.04]
-        exec_model: ${{ fromJson(needs.build_and_cache_Kokkos.outputs.exec_model) }}
-        kokkos_version: ${{ fromJson(needs.build_and_cache_Kokkos.outputs.kokkos_version) }}
-
-    name: Python tests with OpenBLAS, Kokkos and Kokkos Kernels
-    runs-on: ${{ matrix.os }}
-
-    steps:
-      - name: Cancel previous runs
-        uses: styfle/cancel-workflow-action@0.10.0
-        with:
-          access_token: ${{ github.token }}
-
-      - name: Checkout PennyLane-Lightning
-        uses: actions/checkout@v3
-        with:
-          path: main
-          fetch-depth: 2
-
-      - uses: actions/setup-python@v4
-        name: Install Python
-        with:
-          python-version: '3.8'
-
-      - name: Restoring cached dependencies
-        id: kokkos-cache
-        uses: actions/cache@v3
-        with:
-          path: ${{ github.workspace}}/Kokkos_install/${{ matrix.exec_model }}
-          key: ${{ matrix.os }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
-
-      - name: Copy cached libraries
-        run: |
-          mkdir Kokkos/
-          cp -rf ${{ github.workspace}}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
-          pwd
-
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION  libopenblas-dev
-
-      - name: Get required Python packages
-        run: |
-          cd main
-          python -m pip install --upgrade pip
-          pip install -r requirements-dev.txt
-
-      - name: Install ML libraries for interfaces
-        run: |
-          pip install --upgrade torch==$TORCH_VERSION -f https://download.pytorch.org/whl/cpu/torch_stable.html
-          pip install --upgrade "jax[cpu]"  # This also installs jaxlib
-          pip install --upgrade tensorflow~=$TF_VERSION keras~=$TF_VERSION
-
-      - name: Install lightning.qubit device
+      - name: Install all backend devices
+        if: ${{ matrix.pl_backend == 'all' }}pythontestswithBLAS
```

Comment:
Is that `pythontestswithBLAS` normal, after the `if`?

Reply:
Nice catch! It shouldn't be here.